NEWS.rst

```diff
@@ -26,6 +26,10 @@ Fixed typos of triple underscore.
 - Chenghao Zhu
 - Fabian Egli
 
+Store information for 'i' lines in MAF.
+
+- Lucille Lopez-Delisle (first contribution)
+
```

Comment:
The list of names had got split up on the master branch, that is fixed now but will conflict with your addition.